app/lib/intake_pdf.rb

```diff
@@ -136,45 +141,45 @@ def dependents_info
-      spouse_race_black_african_american: bool_checkbox(@intake.demographic_spouse_black_african_american),
-      spouse_race_native_hawaiian_pacific_islander: bool_checkbox(@intake.demographic_spouse_native_hawaiian_pacific_islander),
-      spouse_race_white: bool_checkbox(@intake.demographic_spouse_white),
-      spouse_race_no_answer: bool_checkbox(@intake.demographic_spouse_prefer_not_to_answer_race),
-      ethnicity_hispanic: bool_checkbox(@intake.demographic_primary_ethnicity_hispanic_latino?),
-      ethnicity_not_hispanic: bool_checkbox(@intake.demographic_primary_ethnicity_not_hispanic_latino?),
-      ethnicity_no_answer: bool_checkbox(@intake.demographic_primary_ethnicity_prefer_not_to_answer?),
-      spouse_ethnicity_hispanic: bool_checkbox(@intake.demographic_spouse_ethnicity_hispanic_latino?),
-      spouse_ethnicity_not_hispanic: bool_checkbox(@intake.demographic_spouse_ethnicity_not_hispanic_latino?),
-      spouse_ethnicity_no_answer: bool_checkbox(@intake.demographic_spouse_ethnicity_prefer_not_to_answer?),
+      demographic_english_conversation: @intake.demographic_english_conversation,
+      demographic_english_reading: @intake.demographic_english_reading,
+      demographic_household_disability: @intake.demographic_disability,
+      demographic_household_veteran: @intake.demographic_veteran,
+      demographic_primary_race_american_indian_alaska_native: bool_checkbox_0(@intake.demographic_primary_american_indian_alaska_native),
+      demographic_primary_race_asian: bool_checkbox_0(@intake.demographic_primary_asian),
+      demographic_primary_race_black_african_american: bool_checkbox_0(@intake.demographic_primary_black_african_american),
+      demographic_primary_race_native_hawaiian_pacific_islander: bool_checkbox_0(@intake.demographic_primary_native_hawaiian_pacific_islander),
+      demographic_primary_race_white: bool_checkbox_0(@intake.demographic_primary_white),
+      demographic_primary_race_prefer_not_to_answer_race: bool_checkbox_0(@intake.demographic_primary_prefer_not_to_answer_race),
+      demographic_spouse_race_american_indian_alaska_native: bool_checkbox_0(@intake.demographic_spouse_american_indian_alaska_native),
+      demographic_spouse_race_asian: bool_checkbox_0(@intake.demographic_spouse_asian),
+      demographic_spouse_race_black_african_american: bool_checkbox_0(@intake.demographic_spouse_black_african_american),
+      demographic_spouse_race_native_hawaiian_pacific_islander: bool_checkbox_0(@intake.demographic_spouse_native_hawaiian_pacific_islander),
+      demographic_spouse_race_white: bool_checkbox_0(@intake.demographic_spouse_white),
+      demographic_spouse_race_prefer_not_to_answer_race: bool_checkbox_0(@intake.demographic_spouse_prefer_not_to_answer_race),
+      demographic_primary_ethnicity: @intake.demographic_primary_ethnicity,
+      demographic_spouse_ethnicity: @intake.demographic_spouse_ethnicity,
+    }
+  end
+
+  def signature_info
```

Comment:
@benjaminleesmith See this section -- this is what will add the signature onto this years PDF set for the carryover consent.